components/sidebar/sidebar_channel_button_or_link/sidebar_channel_button_or_link_icon.jsx

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import {Constants} from 'utils/constants.jsx';
+
+import StatusIcon from '../../status_icon.jsx';
+
+export default class SidebarChannelButtonOrLinkIcon extends React.PureComponent {
+    static propTypes = {
+        channelType: PropTypes.string.isRequired,
+        channelId: PropTypes.string.isRequired,
+        membersCount: PropTypes.number,
+        channelStatus: PropTypes.string,
+        teammateId: PropTypes.string,
+        teammateDeletedAt: PropTypes.number
+    }
+
+    render() {
+        var icon = null;
+        const globeIcon = Constants.GLOBE_ICON_SVG;
```

Comment:
This is going to conflict with https://github.com/mattermost/mattermost-webapp/pull/453 when that goes in

Reply:
I'll fix it when the #453 is merged, or we can solve it in the #453, whatever you want.

Reply:
That PR has been merged now, so can you use the new GlobeIcon, LockIcon, and ArchiveIcon components instead of importing these from constants.jsx?

Reply:
Done!